lib/Service/ParticipantService.php

```diff
@@ -1181,6 +1184,11 @@ public function sendCallNotificationForAttendee(Room $room, Participant $current
 			throw new DoesNotExistException('Room mismatch');
 		}
 
+		$userStatus = $this->userStatusManager->getUserStatuses([$attendee->getActorId()]);
+		if (isset($userStatus[0]) && $userStatus[0]->getStatus() === IUserStatus::DND) {
```

Comment:
You can't know that, but the docs of the user status manager are lying. It's actually a map (userid being the key) not a list:
```suggestion
		if (isset($userStatus[$attendee->getActorId()]) && $userStatus[$attendee->getActorId()]->getStatus() === IUserStatus::DND) {
```

Reply:
Docs fixed in https://github.com/nextcloud/server/pull/45219